client/src/pages/locations/Show.tsx

```diff
@@ -268,14 +278,13 @@ const LocationShow = ({ pageDispatchers }: LocationShowProps) => {
                     label="Attachments"
                     component={FieldHelper.ReadonlyField}
                     humanValue={
-                      <div className="attachment-card-list">
-                        {location.attachments.map(attachment => (
-                          <AttachmentCard
-                            key={attachment.uuid}
-                            attachment={attachment}
-                          />
-                        ))}
-                      </div>
+                      <AttachmentsDetailView
+                        attachments={attachments}
+                        updateAttachments={updateAttachments}
+                        relatedObjectType={Location.relatedObjectType}
+                        relatedObjectUuid={values.uuid}
+                        allowEdit={canEdit}
```

Comment:
This is not entirely correct; whether attachment edit is enabled also depends on `Settings.fields.attachment.restrictToAdmins` (see **locations/Form.tsx**).